panel/src/components/Views/ModelView.vue

```diff
@@ -35,16 +33,20 @@ export default {
 			type: Array,
 			default: () => []
 		},
-		uuid: String
+		uuid: String,
+		versions: Object
 	},
 	data() {
 		return {
 			isSaved: true
 		};
 	},
 	computed: {
-		changes() {
-			return this.$panel.content.changes({
+		content() {
+			return this.versions.changes;
```

Comment:
`Suggestion:` Since we are getting rid of the `content` prop and call our whole module `content` (with versions changes and latest), maybe we don't add this computed prop but use `versions.changes` in the templates instead to make it more explicit what version is used.

Reply:
That was all about backward compatibility, but I think you are right that we should rather switch here as well.

Reply:
I looked at it again and we loop the content object through k-sections to each k-section. I think it makes sense to keep the wording here for the sections.